std_msgs/Header header

uint8 led_id

uint8 BLUE=31
uint8 RED=30
uint8 GREEN=32
uint8 YELLOW=33


uint8 BLINK_LEFT=20
uint8 BLINK_RIGHT=21
uint8 BLINK_BLUE=22
uint8 BLINK_RED=23
uint8 BLINK_YELLOW=24
uint8 BLINK_GREEN=25
